### PR TITLE
updated external native Integrations

### PR DESCRIPTION
**Pull request by Aniket-Engg**

### CODE REVIEW

File: docs/foundry.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+Once the correct endpoint is filled in the modal, just click on `OK` and the accounts from the Anvil node will be loaded in the `ACCOUNT` section. Network id will also be shown.
+
+![](images/a-foundry-provider-connected.png)
+
+Now, one can start deploying the contract from Remix IDE to the local Anvil node as usual.
+
+Foundry Remappings
+------------------
+
+Foundry manages dependencies using git submodules and can remap dependencies to make them easier to import. So import defined by remappings can show issue in normal compilation on Remix IDE.
```
Comment (ryestew) [inline docs/foundry.md:28]: Here's an edit - but I'm not totally sure I get your use of "issue":
So imports defined by remappings can show have issues in normal compilation on Remix IDE.

File: docs/foundry.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+To support such compilation, Remix suggests to run compilation using compiler config file. Remix adds some default forge remappings in the compiler config file when a Foundry project is loaded in Remix IDE using remixd.
```
Comment (ryestew) [inline docs/foundry.md:30]: Edit:
To support such compilation, Remix suggests running  compilation using a compiler config file. Remix adds some default **Forge** remappings in the compiler config file when a Foundry project is loaded in Remix IDE using remixd.

File: docs/hardhat.md [hunk 3]
```diff
@@ -72,6 +72,33 @@ Once the correct endpoint is filled in the modal, just click on `OK` and the acc
...
+
+Remix IDE supports hardhat console library while using `Remix VM`. It can be used while making a transaction or running unit tests.
+
+### Deploy and Run Transactions
+
+To try it out, you need to put an import statement and use `console.log` to print the value as shown in image.
+
+![](images/hardhat-console-file.png)
+
+Further, once you execute the `changeOwner` method, value from console statement will be shown in Remix terminal after transaction details as below:
```
Comment (ryestew) [inline docs/hardhat.md:88]: You need to add a "the":
Further, once you execute the `changeOwner` method, the value from ...

File: docs/hardhat.md [hunk 4]
```diff
@@ -72,6 +72,33 @@ Once the correct endpoint is filled in the modal, just click on `OK` and the acc
...
+
+![](images/hardhat-tx-terminal.png)
```
Comment (ryestew) [inline docs/hardhat.md:90]: This image is not so clear - we should update this now or soon maybe with the light theme or just in a larger size.

**Review by ryestew — CHANGES_REQUESTED** [review]

File: docs/foundry.md (hunk 1 above)
Comment (Aniket-Engg) [reply docs/foundry.md:28]: updated

File: docs/foundry.md (hunk 2 above)
Comment (Aniket-Engg) [reply docs/foundry.md:30]: updated

File: docs/hardhat.md (hunk 3 above)
Comment (Aniket-Engg) [reply docs/hardhat.md:88]: updated

File: docs/hardhat.md (hunk 4 above)
Comment (Aniket-Engg) [reply docs/hardhat.md:90]: It has been there for long time. We can update it later.

**Review by ryestew — APPROVED** [review]

**Pull request merged**